lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -100,6 +100,20 @@
                                 options: fieldsData.country.options,
                                 persistChanges: true
                             })
+                        },
+                        {
+                            view: new  AccountSettingsFieldViews.DropdownFieldView({
+                                model: userPreferencesModel,
+                                required: true,
+                                title: gettext('Time Zone'),
+                                valueAttribute: 'time_zone',
+                                helpMessage: gettext(
+                                    'The time zone you currently reside in. ' +
+                                    'This is used to calculate course start/end times.'
```

Comment:
I think it's more than just course start and end times. Perhaps say "... course dates" in keeping with the description in account settings?